model_compression_toolkit/quantizers_infrastructure/pytorch/inferable_quantizers/README.md

```diff
@@ -59,10 +59,44 @@ input_tensor = torch.rand(1, 3, 3, 3).to(device) * 100 - 50
 quantized_tensor = quantizer(input_tensor)
 print(quantized_tensor)
 
-# The maximal threshold is 4 using a signed quantization, so we expect all values to be in this range
+# The maximal threshold is 4.7 using a signed quantization, so we expect all values to be in this range
 assert torch.max(quantized_tensor) < 4.7, f'Quantized values should not contain values greater than maximal threshold'
 assert torch.min(quantized_tensor) >= -4.7, f'Quantized values should not contain values lower than minimal threshold'
 
 ```
 
+The following example demonstrates how to use an activation inferable quantizer:
+
+```python
+# Import PyTorch, Numpy and quantizers_infrastructure
+import numpy as np
+import torch
+
+from model_compression_toolkit import quantizers_infrastructure as qi
+
+# Create an ActivationSymmetricInferableQuantizer quantizer.
+# The quantizer uses 8 bits for quantization and quantizes the tensor per-tensor
+# (per-channel quantization is not supported in activation quantizers).
+# The quantization is unsigned, meaning the range of values is between 0 and the
+# threshold, which is 3.5.
+quantizer = qi.pytorch_inferable_quantizers.ActivationSymmetricInferableQuantizer(num_bits=8,
+                                                                                  threshold=np.asarray([3.5]),
+                                                                                  signed=False)
+
+# Get working device
+device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
+
+# Initialize a random input to quantize between -50 to 50. Move the tensor to the working device
+input_tensor = torch.rand(1, 3, 3, 3).to(device) * 100 - 50
+
+# Quantize tensor
+quantized_tensor = quantizer(input_tensor)
+print(quantized_tensor)
+
+# The quantization is unsigned with threshold of 3.5 - so quantized tensor values should be between 0 to 3.5
+assert torch.max(quantized_tensor) < 3.5, f'Quantized values should not contain values greater than threshold'
+assert torch.min(quantized_tensor) >= 0, f'Quantized values should not contain negative values for unsigned quantization'
+
+```
+
 If you have any questions or issues using the PyTorch inferable quantizers, please open an issue on the GitHub.
```

Comment:
Change this line to "in the issues section in this GitHub repository" (or alternatively, put a link to the page for opening an issue)